### ISSUE TITLE
Error when min config is used as string

### ISSUE THREAD

**Issue by multicolaure** [vlio20/angular-datepicker#595]
Versions: 
- ng2-date-picker : 13.1.1
- Angular : 13.3.6
- Tested on chrome only (but probably not specific)

I am using the `dp-day-time-calendar` component. 

My config object is: 
```js
config = {
  format:  'YYYY-MM-DDTHH:mm:ss',
  min: moment().utc().format('YYYY-MM-DDTHH:mm:ss'), // min date is formatted as string
} as IDatePickerConfig;
```
When selecting a value in the date picker, an error occurs `min.isAfter is not a function` and the selected date is not reflected in the value provided in `ngModel` (although the time is properly changing). 

Changing the format of the min value doesn't fix the issue. 
However, providing a dayjs object fix the issue. 

I am expecting the min value to work with a string, as its type `SingleCalendarValue` accepts strings, and the docs also suggest it is an accepted format. 

Stackblitz: 
https://stackblitz.com/edit/angular-ivy-porqzw?file=src/app/app.component.ts

The same issue exists with `max`.

**Comment by vlio20** [comment]
@multicolaure can you make sure that your format is aligned with the dayjs format?

**Comment by multicolaure** [comment]
I changed the stackblitz to demonstrate the same issue with a format aligned with dayjs : `YYYY-MM-DD[T]HH:mm:ss.SSS[Z]`

The same issue is observed.

**Comment by vlio20** [comment]
Fix in following PR: https://github.com/vlio20/angular-datepicker/pull/601

**Comment by vlio20** [comment]
will be resolved in 14.0.1
